system/defines.php

```diff
@@ -7,25 +7,25 @@
 
 // Directories and Paths
 if (!defined('GRAV_ROOT')) {
-    define('GRAV_ROOT', getcwd());
+    define('GRAV_ROOT', getcwd() . '/');
```

Comment:
GRAV_ROOT was not meant to have trailing slash; trailing slash has been very problematic for me and caused a lot of extra work to remove the flash in many locations. For example locator needs normalized path without trailing slash to work properly.

There's a good rule to remember: always try make minimum possible changes. I would keep the variables as they were defined and just make them deprecated. Otherwise you will easily end up breaking the new or old code instead of just deprecating it.